scss/_badge.scss

```diff
@@ -1,22 +1,32 @@
+// stylelint-disable custom-property-empty-line-before
+
 // Base class
 //
 // Requires one of the contextual, color modifier classes for `color` and
 // `background-color`.
 
 .badge {
+  // scss-docs-start badge-css-vars
+  --#{$variable-prefix}badge-padding: #{subtract($badge-padding-y, 1px) $badge-padding-x add($badge-padding-y, 1px)}; // Boosted mod
+  @include rfs($badge-font-size, --#{$variable-prefix}badge-font-size);
```

Comment:
This line could be at the end of CSS var declarations IMO

Reply:
I'm going to merge as it is in order to avoid having discrepancies with Bootstrap (for easier merges in the future). But if the fix is OK, you could propose it to Bootstrap so we have it back with the next merges.